### PR TITLE
Helpers creation

### PR DESCRIPTION
**Pull request by FacundoPucheta**
New helpers added(4) on (/src/helpers):

- isInsidePolygon.js
- standardE30_2003.js | with small fixed value on lines 51 - 54.
- standardE30_2015.js 
- standardE30_2015Spec.js

### CODE REVIEW

File: src/helpers/isInsidePolygon.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+for (let i = 0, j = nVert - 1; i < nVert; j = i++) {
+
+    if(helper) c = !c ;
+
+  };
+  
+return c;
+
+
```
Comment (manduinca) [inline src/helpers/isInsidePolygon.js:16]: Why three empty lines? I think one line is ok.

File: src/helpers/e30_2003.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const e30_2003 = async (location, groundType) => {
+
+  const location_data = await Location.findOne({
+    where: { id: location },
+  });
+  const lat = location_data.latitude;
+  const long = location_data.longitude;
+
+  const polygonsFound = await Polygon.findAll({
```
Comment (manduinca) [inline src/helpers/e30_2003.js]: `foundPolygons`?

File: src/helpers/e30_2003.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    where: { type: 3 },
+  });
+  let polygons = [];
+  for (const p of polygonsFound) {
+    polygons.push(p.points.split("|"));
+  }
+
+  const coordinates = await Point_polygon.findAll({
```
Comment (manduinca) [inline src/helpers/e30_2003.js]: maybe: `raw_coordinates`

File: src/helpers/e30_2003.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    where: { type: 3 },
+  });
+  let coordenadas = [];
```
Comment (manduinca) [inline src/helpers/e30_2003.js]: maybe: `coordinates`

**Review by manduinca — DISMISSED** [review]

**Review by manduinca — DISMISSED** [review]
Only `interpolar` is in spanish.

**Review by manduinca — APPROVED** [review]
LGTM!

**Review by joser098 — APPROVED** [review]

**Pull request merged**